tests/test_status_filters.py

```diff
@@ -0,0 +1,73 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.treeherder import TreeherderPage
+
+
+@pytest.mark.nondestructive
+def test_nav_status_filters(base_url, selenium):
+    """Filter jobs using status filters in the nav"""
+    page = TreeherderPage(selenium, base_url).open()
+
+    """Remove all nav filters"""
+    page.filter_job_failures()
+    page.filter_job_successes()
+    page.filter_job_retries()
+    page.filter_job_usercancel()
+    page.filter_job_in_progress()
+    all_jobs = len(page.all_jobs)
```

Comment:
This particular test was counting to make sure that zero jobs appear- so len is correct. As @davehunt pointed out in another comment the list() function is built in and can be removed.